custom_components/tuya_local/devices/Fresco_Hydrate_Ultra_Pet_Fountain.yaml

```diff
@@ -0,0 +1,111 @@
+name: Hydrate Ultra
+products:
+  - id: cuw2k3ho3e25fade
+    name: Hydrate Ultra
+
+primary_entity:
+  entity: light
+  icon: "mdi:water-pump"
+  dps:
+    - id: 101
```

Comment:
Is this really a switch for the light only?

If it is for the whole device, then it might be better to separate it into a separate switch.

Reply:
The light is together with the on switch, and you cant add is separate.
The switch is for on off for device, and merging the brightness with the on off button works good and looks nicer.
I bet 90% of the people they don't care about the led anyway.